yarn.lock

```diff
@@ -3095,44 +3078,49 @@
+  version "2.20.0"
+  resolved "https://registry.yarnpkg.com/@sentry-internal/rrweb-player/-/rrweb-player-2.20.0.tgz#e14a129dce9b0908863c736dacabcca71e000c3b"
+  integrity sha512-l+hCYMaUoUDI/t2bQfkZecz9g73TwfrDOdYB3YyZlfKqoWwtVugieSI2Y5YiMjNFvwSyYjxxr5vJ0AFC6LDJFA==
   dependencies:
-    "@sentry-internal/rrweb" "2.18.0"
+    "@sentry-internal/rrweb" "2.20.0"
     "@tsconfig/svelte" "^1.0.0"
 
-"@sentry-internal/rrweb-snapshot@2.18.0":
-  version "2.18.0"
-  resolved "https://registry.yarnpkg.com/@sentry-internal/rrweb-snapshot/-/rrweb-snapshot-2.18.0.tgz#5dd348ac2d27eb717328add0112a5e463eb3afc0"
-  integrity sha512-HeyR/WZarWZIFya+6HeAfe7Fz5RF0ehVaBrKB3F08TqbD8pk1UHn6FFV4irB/vOUjcChJzKMZQ6rs2TlsjJjVg==
+"@sentry-internal/rrweb-snapshot@2.20.0":
+  version "2.20.0"
+  resolved "https://registry.yarnpkg.com/@sentry-internal/rrweb-snapshot/-/rrweb-snapshot-2.20.0.tgz#005be8b5a2da4a96f7a863dbed3eccc1c68bfd1f"
+  integrity sha512-T+KTcOWChgaYfAL0PCUv7PQuMI2k9BG+3X8IlVPgb2/NTPKvJYicHxwC7tcwRM5fCqZpf2Erohyb1R6SuLJJPA==
+  dependencies:
+    downlevel-dts "https://github.com/getsentry/downlevel-dts"
 
-"@sentry-internal/rrweb-types@2.18.0":
-  version "2.18.0"
-  resolved "https://registry.yarnpkg.com/@sentry-internal/rrweb-types/-/rrweb-types-2.18.0.tgz#97706642d86e2500e2fd31a8fb72a9c7a21f4d60"
-  integrity sha512-5WRVxz+jcUB/xo+m4vipcm6VbDbQtRL/l+8PusFAz8GuCwN5nR7VoNQU4uNbwrWcVg/U6rLi4cY3Rdb2e3mfxw==
+"@sentry-internal/rrweb-types@2.20.0":
+  version "2.20.0"
+  resolved "https://registry.yarnpkg.com/@sentry-internal/rrweb-types/-/rrweb-types-2.20.0.tgz#4e4531edf68c537a791cf0a44daa9bc3ca1e98ae"
+  integrity sha512-0LZDglrvCjCdyk9Uc1sJPIdS9IdQEDrfB0S/vC3EaaPteUfdsp4CK4yaXn1cCatbb75Uev5tIICtPZkq9zzqkg==
   dependencies:
-    "@sentry-internal/rrweb-snapshot" "2.18.0"
+    "@sentry-internal/rrweb-snapshot" "2.20.0"
+    "@types/css-font-loading-module" "0.0.7"
+    downlevel-dts "https://github.com/getsentry/downlevel-dts"
 
-"@sentry-internal/rrweb@2.18.0":
-  version "2.18.0"
-  resolved "https://registry.yarnpkg.com/@sentry-internal/rrweb/-/rrweb-2.18.0.tgz#3726c5af0d41a352f7314efb20406a64b0691a71"
-  integrity sha512-FV+ZcndQyHkYAUB+1MChkINLS5Y6K+GIQ5WPFNSbue1LT98tCB6z0sMzVy5Z6TOCTl5GIaMcDwqwI8L96PmlIg==
+"@sentry-internal/rrweb@2.20.0":
+  version "2.20.0"
+  resolved "https://registry.yarnpkg.com/@sentry-internal/rrweb/-/rrweb-2.20.0.tgz#5f8034542c2562da40b875516762afd04d0d5238"
+  integrity sha512-6uMZwDDVUtBy8KTegCM9IYUc91yHNJ87o7GMdUpBImPOQqmMt9HsHJl5vur/KBnPFTiAUNeRf1Ar8iMH62auQA==
   dependencies:
-    "@sentry-internal/rrdom" "2.18.0"
-    "@sentry-internal/rrweb-snapshot" "2.18.0"
-    "@sentry-internal/rrweb-types" "2.18.0"
+    "@sentry-internal/rrdom" "2.20.0"
+    "@sentry-internal/rrweb-snapshot" "2.20.0"
+    "@sentry-internal/rrweb-types" "2.20.0"
     "@types/css-font-loading-module" "0.0.7"
     "@xstate/fsm" "^1.4.0"
     base64-arraybuffer "^1.0.1"
+    downlevel-dts "https://github.com/getsentry/downlevel-dts"
```

Comment:
sentry-internal/rrweb should probably move downlevel-dts to dev dependencies, doesn't seem like something everyone needs to install

Reply:
cc @billyvg

Reply:
https://github.com/getsentry/rrweb/pull/204